src/docs/development/frameworks-web/angular.mdx

```diff
@@ -126,12 +125,12 @@ PX Blue provides packages for code standards and style using ESLint and Prettier
 
 ## Browser Support
 
-Angular and Angular Material will work with any modern browser. For specific details, you can view their official support pages:
```

Comment:
Supports Angular and Angular Material... not React.

Reply:
haha nothing like copy/paste